### ISSUE TITLE
Overriding _settings.scss 

### ISSUE THREAD

**Issue by scottmcpherson** [lgollut/meteor-foundation5-sass#4]
Hey @lgollut, one thing I like to do when working with foundation 5 projects is to un-comment and modify a few of the settings in the _settings.scss. Because the _settings.scss file is located in the package directory, I don't really want to mess with it. How do you recommend changing the settings? Should I just copy the entire _settings.scss file into my scss directory, or will the preprocessor have compile that file twice?

I guess the obvious thing to do would be to create a separate settings file in my scss dir, and just copy the settings that I need to change into that file. I just kind of like having that entire settings file in my scss dir.

**Comment by kylesuss** [comment]
+1. Came here looking for this. Would be happy to help implement if we can come up with a mutually agreeable decision.

My first thought is that _foundation.scss in the foundation5-sass directory should not import the settings file; rather, that should be done in our personal SCSS file. So the readme would look something like this:

### Add the scss file

Add the following to your main .scss file:

```
@import "settings";
@import "packages/foundation5-sass/foundation";
```

Feedback?

**Comment by scottmcpherson** [comment]
@kylesuss I agree. That's actually what I did yesterday with a project that I started. I couldn't really see any way around doing it like that. I already forked, cloned, and made the changes yesterday but I don't think I've pushed the up to GH yet.

**Comment by bjesus** [comment]
Any news on this? is there a way to achieve this?
